resources/model/OpenStudio.idd

```diff
@@ -11857,6 +11891,15 @@ OS:AirTerminal:DualDuct:VAV,
        \note be computed based on the current number of occupants in the zone.
        \note At no time will the supply air flow rate exceed the value for Maximum Air Flow Rate.
        \note If this field is blank, then the terminal unit will not be controlled for outdoor air flow.
+  A8 ; \field Minimum Air Flow Turndown Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note This field adjusts the design minimum flow rate by multiplying it using this schedule of fraction
+       \note values. This field is used with "Zone Minimum Air Flow Fraction". Schedule values are fractions
+       \note 0.0 to 1.0. This field adjusts the minimum airflow turndown below the design minimum air flow
+       \note and is intended for use with ASHRAE Standard 170. If this field is left blank, then the turndown
+       \note minimum air flow fraction value is set to 1 and the VAV air terminal uses fixed fraction specified
+       \note in the field "Zone Minimum Air Flow Fraction".
```

Comment:
Ok, so this is the last field added in all objects, so unless it's required-field, no VT is expected to happen.